docs/topics/proc_deploying-the-cache-booster-using-the-oc-cli-client.adoc

```diff
@@ -76,7 +77,7 @@ cache-server-123456789-aaaaa             1/1       Running     0          8m
 +
 Your 3 pods should have a status of `Running` once they are fully deployed and started.
 
-. Once your booster is deployed and started, determine its route.
+. Once your example is deployed and started, determine its route.
```

Comment:
Once your example is deployed and started, determine its route.--> After your example application is deployed and started, determine its route.

Reply:
Thanks. I updated the parallel modules as well.